utils/driverless.c

```diff
@@ -554,7 +556,14 @@ list_printers (int mode, int reg_type_no, int isFax)
 
     while ((bytes = cupsFileGetLine(fp, buffer, sizeof(buffer))) > 0) 
     {
-      ippfind_output = (char *)malloc(8192*(sizeof(char)));
+      ippfind_output = (char *)malloc(MAX_OUTPUT_LEN*(sizeof(char)));
+      if (ippfind_output == NULL)
+      {
+        exit_status = 1;
+        goto error;
+      }
+
+      ptr = buffer;
       ptr = buffer;
```

Comment:
Why `ptr = buffer;` twice here?

Reply:
Sorry, copy/paste error (+1 error in vim copy/paste :D ). Fixed now.